packages/browser-sdk/src/lib/embed/index.ts

```diff
@@ -227,6 +228,9 @@ define("WherebyEmbed", {
     endMeeting() {
         this._postCommand("end_meeting");
     },
+    leaveMeeting() {
+        this._postCommand("room_left");
```

Comment:
(Second half of the comment [here](https://github.com/whereby/pwa/pull/4078#discussion_r1588833129).) This should be updated to `leave_room`:

```suggestion
        this._postCommand("leave_room");
```

Reply:
Fixed in latest commit. Thanks!